setup/README.md

```diff
@@ -0,0 +1,74 @@
+# SteamPulse
+
+An integrated dashboard displaying trend analysis for new releases on steam
+
+## Configure environment
+
+```sh
+python3 -m venv venv
+source ./venv/bin/activate
+pip3 install -r requirements.txt
+```
+
+## Configure environment variables
+
+The following environment variables must be supplied in a `.env` file in the root directory, and a `terraform.tfvars` file in the setup directory:
+
+```sh
+ACCESS_KEY_ID     = XXXXXXXXXXX
+SECRET_ACCESS_KEY = XXXXXXXXXXX
+DATABASE_NAME     = steampulse
+DATABASE_USERNAME = steampulse_admin
+DATABASE_PASSWORD = XXXXXXXXXXX
+DATABASE_ENDPOINT = XXXXXXXXXXX
+```
+
+If hosting this service on AWS, you'll need to create your RDS to get the database endpoint.
+
+## Setup cloud resources
+
+**Warning** - AWS can incur unexpected costs, be sure you know what you're doing before replicating this section.
+
+0. If you haven't already, setup [terraform](https://developer.hashicorp.com/terraform/downloads) and [docker](https://docs.docker.com/engine/install/).
+
+1. Navigate to the setup directory.
+
+2. `terraform init` to initalise the directory with terraform setup files.
+
+3. `terraform apply` to construct AWS resources.
+
+4. TODO(I haven't set this up yet, but we output should give us the db endpoint) -> add it to your env file.
```

Comment:
I thought the idea was that they would be accessing our database -- but this works too